commercial/app/model/commercial/events/LiveEventMembershipInfo.scala

```diff
@@ -0,0 +1,22 @@
+package model.commercial.events
+
+import play.api.data.validation.ValidationError
+import play.api.libs.json._
+
+case class LiveEventMembershipInfo(id: String,
+                                   url: String,
+                                   mainImageUrl: String)
+
+object LiveEventMembershipInfo {
+  implicit val format = Json.format[LiveEventMembershipInfo]
+
+  // based on play.api.libs.json.LowPriorityDefaultReads.traversableReads
+  implicit val formats = new Reads[Seq[LiveEventMembershipInfo]] {
+    override def reads(json: JsValue): JsResult[Seq[LiveEventMembershipInfo]] = {
+      json match {
+        case JsArray(jsValues) => JsSuccess(jsValues.flatMap(_.asOpt[LiveEventMembershipInfo]))
+        case _ => JsError(Seq(JsPath() -> Seq(ValidationError("error.expected.jsarray containing LiveEventMembershipInfo"))))
```

Comment:
This is a pattern we've used before to tolerate the presence of a malformed element.
1. We expect it to be a `JsArray`, if it isn't then error out
2. If it is a JsArray then try to parse each element - either we get a `Some[LiveEventMembershipInfo]` or we get a `None` at which point we can flatMap it.